src/Migration.php

```diff
@@ -198,7 +198,7 @@ public function log($message, $warning)
        // Do not log if more than 3 log error
         if (
             $this->log_errors < 3
-            && !Toolbox::logInFile($log_file_name, $message . ' @\n', true)
+            && !Toolbox::logInFile($log_file_name, $message . "\n", true)
```

Comment:
Format should probably be preserved.
```suggestion
            && !Toolbox::logInFile($log_file_name, $message . "@ \n", true)
```

Reply:
What does the @ at the end of the line mean?

Reply:
I guess migration log files format is not so important, and it looks like it was anyway not really consistent.

Previous format was:
```
2023-07-26 10:20:54 [@bd2e74b91674]
1 second for "Work in progress..." @ 2023-07-26 10:20:54 [@bd2e74b91674]
0 seconds for "Data migration - glpi_displaypreferences" @ 2023-07-26 10:20:54 [@bd2e74b91674]
0 seconds for "Task completed." @ 2023-08-01 13:51:17 [2@bd2e74b91674]
...
```

With removal of this `@` and with a `\n`, it should look like:
```
2023-07-26 10:20:54 [@bd2e74b91674] 1 second for "Work in progress..."
2023-07-26 10:20:54 [@bd2e74b91674] 0 seconds for "Data migration - glpi_displaypreferences"
2023-07-26 10:20:54 [@bd2e74b91674] 0 seconds for "Task completed."
2023-08-01 13:51:17 [2@bd2e74b91674] ...
```

It seems indeed better.